### PR TITLE
Make ndb pass lint tests

### PR DESCRIPTION
**Pull request by pcostell**
This includes:
* Fixing hanging indents. I don't see anything along these lines in style guides, but I can't get pylint to not complain about it.
* Whitespaces with operators.

### CODE REVIEW

**Comment by pcostell** [comment]
@eddavisson -- another review for you.

File: pylintrc [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+# RATIONALE: import statements are cleaner on a single line.
+# DEFAULT: ^\s*(# )?<?https?://\S+>?$
+ignore-long-lines=^(\s*(# )?<?https?://\S+>?)|(from .* import .*)$
+
+# Standard indent.
+# DEFAULT: '    '
+# RATIONALE: Google style dictates 2 spaces, not 4.
+indent-string='  '
+
+# RATIONAL: Match google style guide instead of default.
```
Comment (eddavisson) [inline pylintrc]: RATIONALE, s/google/Google/

File: pylintrc [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+# of functions or classes which do not require a docstring.
+no-docstring-rgx=(__.*__|main)
+
+# Min length in lines of a function that requires a docstring.
+docstring-min-length=10
+
+# Regular expression which should only match correct module names. The
+# leading underscore is sanctioned for private modules by Google's style
+# guide.
+#
```
Comment (eddavisson) [inline pylintrc]: extra # line

**Comment by eddavisson** [comment]
LGTM

**Pull request merged**